lib/scrub/chunker.rb

```diff
@@ -9,31 +10,47 @@ module Scrub
   # #{chunk_count} equal-sized (roughly) files from them.
   class Chunker
     attr_reader :glob, :chunks
-    def initialize(glob, chunk_count = 16)
+    def initialize(glob: "/dev/null", chunk_count: 16, add_uuid: false, out_ext: nil)
```

Comment:
Under what circumstances would we call `Chunker.new` with no glob? Likewise, we check that `out_ext` isn't nil on line 34, so could we just make that a required parameter?

Reply:
good point. done